baton/modules/twilio/twilio.go

```diff
@@ -0,0 +1,179 @@
+package twilio
+
+import (
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/gorilla/mux"
+)
+
+type Twilio struct {
+	UserId, ApiKey string
+}
+
+var (
+	smsCallbacks  = make([]callback, 0)
+	callCallbacks = make([]callback, 0)
+)
+
+type callback struct {
+	number string
+	resp   chan<- interface{}
+	data   string
+}
+
+var client = &http.Client{}
```

Comment:
What if instead of making the HTTP requests directly from this package, we either separated it into a separate package or used an existing Twilio binding (like https://github.com/sfreiberg/gotwilio)?